### PR TITLE
Add server boot script

### PR DESCRIPTION
**Pull request by JoakimLandberg**
Adds a script with common code for server start and stop functions.
These functions are now used in the deploy and rollback scripts.

Also used the common code for extract in the deploy script.

### CODE REVIEW

**Comment by JoakimLandberg** [comment]
Scripts tested locally

File: init-mt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  mtVersion=`echo $workDir | sed -e 's/.*-\(.*\)/\1/g'`
+  command="forever --killSignal SIGINT -a -l $projectName-$mtVersion.log start $SCRIPT --conf $ENV --credentials $credentialsFilePath"
+  /bin/su -s /bin/bash -c "$command" $USER
+  /bin/su -s /bin/bash -c "forever list" $USER
+}
+
+function stopServer {
+  echo "Stopping Mediatool"
+  cd $DIRECTORY
+  command="whoami && forever list && forever stop $SCRIPT"
```
Comment (TheLudd) [inline init-mt]: We can skip the `whoami` part. It was really only used for debugging in the early days

**Comment by TheLudd** [comment]
So this fixes the rollback script right? We should also fix the `init.d` file. It is not versioned yet. But it could be put in this repository and then linked from the `init.d` folder.

**Comment by JoakimLandberg** [comment]
@TheLudd Yes it will now be safe to rollback again. We can add an additional script for init.d.

**Pull request merged**